aten/src/ATen/gen.py

```diff
@@ -351,7 +364,7 @@ def generate_storage_type_and_tensor(backend, density, scalar_type, declarations
     fm.write(env['Type'] + ".h", TYPE_DERIVED_H, env)
 
     type_register = TYPE_REGISTER.substitute(backend=env['Backend'], scalar_type=scalar_name, type_name=env['Type'])
-    if env['DenseBackend'] == 'CPU':
+    if 'CPU' in env['DenseBackend']:
```

Comment:
did this need to change?  I thought this should just be DeviceType-based (see above?)

Reply:
yeah probably should revert this change.

Reply:
Do you mean change 'DenseBackend' to 'DeviceType' instead?